src/App.tsx

```diff
@@ -1,5 +1,49 @@
+import colors from './colors.json';
+
+function ColorObject({ colorInfo }) {
+    return (
+        <div
+            style={{
+                color: `#${colorInfo.hex}`,
+            }}
+        >
+            {colorInfo.name}
+        </div>
+    );
+}
+
+function ColorObjectWithComp({ colorInfo }) {
+    return (
+        <div style={{ display: 'flex', gap: '12px' }}>
+            <div style={{ width: '128px' }}>
+                <ColorObject colorInfo={colorInfo} />
+            </div>
+            <div>Complimentary: </div>
+            <div style={{ display: 'flex', gap: '12px' }}>
+                {colorInfo.comp.map(i => {
```

Comment:
Here as well: use more meaningful names for naming. It's difficult to understand just by reading the code  what `i` is. You could name it `complimentaryColor`.